BundID/Views/SetupScan.swift

```diff
@@ -88,7 +100,9 @@ extension SetupScanState {
             }
             
             // Wrong transport/personal PIN provided
-            if self.remainingAttempts != remainingAttempts {
+            if let remainingAttemptsBefore = self.remainingAttempts,
+               remainingAttempts < remainingAttemptsBefore {
+                self.remainingAttempts = remainingAttempts
```

Comment:
Sollte nicht in jedem Fall die `self.remainingAttempts = remainingAttempts` gesetzt werden, damit wir beim nächsten Mal wissen, wie viel wir erwarten? Also falls der Ausweis von 2 verbleibenden Versuchen auf 3 springt.

Vielleicht merken wir uns ganz oben einmal die `remainingAttemptsBefore` und setzen self.remainingAttempts direkt mal auf den neuen Wert?